meshtastic/mesh_interface_runtime/__init__.py

```
@@ -0,0 +1,27 @@
+    VALID_TELEMETRY_TYPE_SET,
+    VALID_TELEMETRY_TYPES,
+    TelemetryType,
+)
+
+__all__ = [
+    # Telemetry types and constants - used by mesh_interface.py
+    "TelemetryType",
+    "DEFAULT_TELEMETRY_TYPE",
+    "VALID_TELEMETRY_TYPES",
+    "VALID_TELEMETRY_TYPE_SET",
+]
```

Ruff (0.15.7)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)